JoinColony/colonyCDapp
# fix: permissions sidebar margins and cleanup sidebar stuff a bit

## Description

The entire solution was that the `UserPopover` was wrapped in a `div` and that increased the height.
But as I was perusing the code I saw a lot of remnants regarding fetching the action in the form, since we used to reuse the form for that, and I cleaned some of it up.
The `ActionSidebarContent` never rendered if `transactionId` was defined, since we showed a loader or the `CompletedAction` component.

[Figma link](https://www.figma.com/design/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?node-id=3701-19297&t=dzGUdgFPVEu8YEVa-4)
![image](https://github.com/user-attachments/assets/d281318d-fe97-4fed-a529-fd30c30100d6)


## Testing

1. Create an action using `Permissions` and assure that the margins and alignment now matches Figma
2. Verify that creating an advanced payment with `Staking` as the decision method still works correctly (so that the first modal with the amount and fees opens)
3. If you think of any case you can try that one too

## Diffs

**Changes** 🏗

* Fixed permissions sidebar padding
* `PermissionsSidebar` now receives the actions as a property since it doesn't ever refetch/poll it

**Deletions** ⚰️

* `ActionSidebarContent` doesn't use `isMotion` and `transactionId` anymore, also removed the rendering of the sidebar


Resolves  #2959
